### PR TITLE
Update shunt documentation + regulation documentation in general

### PR DESCRIPTION
**Pull request by MioRtia**
Signed-off-by: RALAMBOTIANA MIORA <miora.ralambotiana@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**
Doc update. The regulation shunt is now documented (cf PR powsybl/powsybl-core#958) and some corrections have been done on RTC and PTC documentation.

### CODE REVIEW

File: docs/iidm/model/shuntCompensator.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ The `com.powsybl.iidm.network.ShuntCompensator` interface is used to model a shu
 | bPerSection | double | S | yes | - | The Positive sequence shunt (charging) susceptance per section |
 | MaximumSectionCount| integer | int | yes | - | The maximum number of sections that may be switched on |
 | CurrentSectionCount | integer | int | yes | - | The current number of section that may be switched on |
+| RegulatingTerminal | [`Terminal`](terminal.md) | - | no | The shunt compensator's terminal | The terminal used for regulation |
+| TargetV | double | kV | only if `VoltageRegulatorOn` is set to `true` | - |  The voltage target |
+| TargetDeadband | double | kV | only if `VoltageRegulatorOn` is set to `true` | - | The deadband used to avoid excessive update of controls |
+| VoltageRegulatorOn | boolean | - | no | false | The voltage regulating status |
 
 ## Section
 A section of a shunt compensator is an individual capacitor or reactor.
```
Comment (annetill) [inline docs/iidm/model/shuntCompensator.md]: Can you add maybe a sentence to explain that this regulation could be used or not by power flows to model a human action to increase or decrease a local volatge plan ?

Comment (MioRtia) [reply docs/iidm/model/shuntCompensator.md]: Done (below)

**Comment by annetill** [comment]
That's great, I am going to merge this PR, do you agree ?

**Comment by MioRtia** [comment]
> That's great, I am going to merge this PR, do you agree ?

Yes, fine for me

**Pull request merged**